src/reference/asciidoc/kafka.adoc

```diff
@@ -126,9 +126,57 @@ Note, however that flushing will likely significantly reduce performance.
 
 ==== Receiving Messages
 
-Messages can be received by configuring a `MessageListenerContainer` and providing a `MessageListener`, or by
+Messages can be received by configuring a `MessageListenerContainer` and providing a <<message-listener, Message Listener>>, or by
```

Comment:
Well, not yet.

This link is a bug.
We have a tag as `message-listeners`, but OTOH it is just a next paragraph, so may be there is just no need in the link.